bigframes/session/_io/bigquery/read_gbq_table.py

```diff
@@ -162,21 +162,21 @@ def get_ibis_time_travel_table(
 
 def are_index_cols_unique(
     bqclient: bigquery.Client,
-    ibis_client: ibis.BaseBackend,
     table: bigquery.table.Table,
     index_cols: List[str],
     api_name: str,
 ) -> bool:
+    if len(index_cols) == 0:
```

Comment:
Let's add some unit tests specificaly for this method with `index_cols = []` and `primary_keys = []; index_cols = ['some_col']`

Reply:
added some unit tests